src/components/Form/Documents/DownloadPdf.tsx

```diff
@@ -0,0 +1,51 @@
+import { Grid, Typography } from "@mui/material";
+import DropBox from "./DropBox";
+import DownloadIcon from "@mui/icons-material/Download";
+const documentSteps = [
```

Comment:
documentSteps = ["1. Sprawdź wprowadzone dane i pobierz dokument z wypełnioną ankietą", "2. Wydrukuj swoją ankietę i podpisz w wyznaczonym polu", "3.Załącz skan podpisanego dokumentu"]

and then:
 {documentSteps.map((step) => (
  <Grid item xs={12}>
            <Typography sx={{ color: "primary.main", mt: 1 }}>
              {step}
            </Typography>
  </Grid>,
          ))}

it will be shorter